submissions/egwoloOkorodudu/HTML/product.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```

Comment:
You need to style the already created product page. You are not meant to create a new file for product page.@Egwolo2

Reply:
I think that was what i did.  I only changed the name from index.html to product.html

Reply:
ok